### DISCUSSION TITLE
Disabled navigation arrows

### DISCUSSION THREAD

**Discussion by tehnolog** [Valexr/Slidy#24] [Ideas]
Is there any way to show default (not external) arrows in "disabled" state? Arrows are being removed from DOM when slider reach start/end.

Maybe better solution is to add "disabled" attribute to the arrows and leave ability to hide them just with CSS.

**Comment by Valexr** [comment]
May be you are right🤓but I think it right behaviour for not looped carousel... in `loop` mode arrows visible 
How you see implemented `disabled` flag?
